### DISCUSSION TITLE
Tool for making lab handbooks

### DISCUSSION THREAD

**Discussion by Aariq** [SORTEE-Github-Hackathon/manuscript#240] [General]
Just came across this on Twitter: https://github.com/very-good-science/our-handbook-user-guide

A template for making lab handbooks with Jupyter.  Would be cool to port this to Quarto once there's a stable release!

**Comment by Aariq** [comment]
Ah, this is maybe a better link: https://very-good-science.github.io/our-handbook/guide/what-is-our-handbook.html

**Reply by wavingtowaves** [reply]
Oh, that's awesome. Thanks for sharing @Aariq!
